sdk/java/pulumi/src/main/java/com/pulumi/core/function/ServiceProvider.java

```diff
@@ -1,7 +0,0 @@
-package com.pulumi.core.function;
-
-// FIXME: some core c# interface, will probably not be needed, not in this form at lest,
-//        looks like some kind of injection
-public interface ServiceProvider {
```

Comment:
an old misunderstanding while porting initial PoC, somehow survived this long ;)